Sources/ConsoleKit/Utilities/GenerateAutocompleteCommand.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+struct GenerateAutocompleteCommand: Command {
+
+    var help: String { "Generate shell completion scripts for the executable" }
+
+    var rootCommand: AnyCommand?
+
+    init(rootCommand: AnyCommand? = nil) {
+        self.rootCommand = rootCommand
+    }
+
+    struct Signature: CommandSignature {
+
+        @Argument(
+            name: "shell",
+            help: "The shell to target (\(Shell.allCases.map { "\($0)" }.joined(separator: "|")))",
+            completion: .values(of: Shell.self)
+        )
+        var shell: Shell
+    }
+
+    func run(using context: CommandContext, signature: Signature) throws {
+        guard let rootCommand = self.rootCommand else { fatalError("`rootCommand` was not initialized") }
```

Comment:
Since `GenerateAutocompleteCommand` is `internal`, anytime it's actually _run_—as opposed to simply being used to generate autocompletion—we know the `rootCommand` should have been set.